cmd/argo/commands/clustertemplate/util.go

```diff
@@ -0,0 +1,31 @@
+package clustertemplate
+
+import (
+	wfv1 "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
+	"github.com/argoproj/argo-workflows/v3/workflow/util"
+	"log"
+	"os"
+)
+
+func generateClusterWorkflowTemplates(filePaths []string, strict bool) []wfv1.ClusterWorkflowTemplate {
+	fileContents, err := util.ReadManifest(filePaths...)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var clusterWorkflowTemplates []wfv1.ClusterWorkflowTemplate
+	for _, body := range fileContents {
+		cwftmpls, err := unmarshalClusterWorkflowTemplates(body, strict)
+		if err != nil {
+			log.Fatalf("Failed to parse cluster workflow template: %v", err)
+		}
+		clusterWorkflowTemplates = append(clusterWorkflowTemplates, cwftmpls...)
+	}
+
+	if len(clusterWorkflowTemplates) == 0 {
+		log.Println("No cluster workflow template found in given files")
+		os.Exit(1)
```

Comment:
oh just realized this could be simplified too:
```suggestion
		log.Fatalln("No cluster workflow template found in given files")
```